cmd/wallet-js-sdk/src/did/didmanager.js

```diff
@@ -5,9 +5,13 @@ SPDX-License-Identifier: Apache-2.0
 */
 
 import { contentTypes, getMediatorConnections, UniversalWallet } from "..";
+import {expect} from "chai";
+import {testConfig} from "../../test/specs/common";
```

Comment:
why are you importing test code in SDK package??? test code never gets bundled. You will package bundle will fail in wallet.

Reply:
removed, these were added when testing stuff, but forgot to remove them